.github/dependabot.yaml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: gradle
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: sunday
+  - package-ecosystem: "github-actions"
```

Can we use the same syntax as we do above?

```suggestion
  - package-ecosystem: github-actions
```